### DISCUSSION TITLE
General Contribution guideline.

### DISCUSSION THREAD

**Discussion by WaliNeslIt** [btcpayserver/btcpayserver#5935] [General]
Hi, I want to contribute in this project, I saw the code and is understandable but I don't know the flow of the application. Can somebody guide how to start and understand the flow so that I can take part in the contribution. Any links, documentation, videos & blogs must be appreciated.

**Comment by pavlenex** [comment]
Where exactly did you look? It's literary in the readme https://github.com/btcpayserver/btcpayserver?tab=readme-ov-file#-contributing

**Reply by WaliNeslIt** [reply]
thanks

**Comment by WaliNeslIt** [comment]
done
